apps/omg_performance/test/omg_performance/integration/byzantine_event_test.exs

```diff
@@ -0,0 +1,80 @@
+# Copyright 2019 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.Performance.ByzantineEventsTest do
+  use ExUnitFixtures
+  use ExUnit.Case, async: false
+  use OMG.ChildChain.Integration.Fixtures
+  use OMG.Watcher.Fixtures
+
+  alias OMG.Performance
+  alias OMG.Performance.ByzantineEvents
+  alias OMG.Performance.ByzantineEvents.Generators
+
+  @moduletag :integration
+  @moduletag timeout: 180_000
+
+  setup_all do
+    Application.put_env(:omg_child_chain, :mix_env, "dev")
+    Application.put_env(:omg_watcher, :mix_env, "dev")
+
+    on_exit(fn ->
+      Application.put_env(:omg_child_chain, :mix_env, nil)
+      Application.put_env(:omg_watcher, :mix_env, nil)
+    end)
+
+    :ok
+  end
+
+  @tag fixtures: [:contract, :child_chain, :omg_watcher]
+  test "time response for asking for exit data", %{contract: %{contract_addr: contract}} do
+    dos_users = 3
+    ntx_to_send = 100
+    spenders = Generators.generate_users(4)
+    exit_per_dos = length(spenders) * ntx_to_send
+    total_exits = length(spenders) * ntx_to_send * dos_users
+
+    IO.puts("""
```

Comment:
don't IO.puts in the test. Do some assertions that smoke test the perf test instead, for this to be useful